ambry-quota/src/main/java/com/github/ambry/quota/capacityunit/CapacityUnit.java

```diff
@@ -16,7 +16,7 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.github.ambry.quota.QuotaName;
 import java.util.concurrent.atomic.AtomicLong;
-import org.codehaus.jackson.annotate.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
QQ: any reason to change this?

Reply:
code doesn't compile without this

Reply:
Jing's PR https://github.com/linkedin/ambry/pull/2123/files#diff-63fa9099c720e34c848596bd8c1eb05ecf92da8810196b579ef47611fa1bc15bR206

Reply:
the change is good, we shouldn't be using codehaus library anymore. fasterxml library is the newer release for the same library.